question-service/app/routers/questions.py

```diff
@@ -31,3 +28,11 @@ async def create(question: QuestionModel):
 @router.get("/", response_description="Get all questions", response_model=QuestionCollection)
 async def get_all():
     return await get_all_questions()
+
+@router.get("/{question_id}", response_description="Get question with specified id", response_model=QuestionModel)
+async def get_question(question_id: str):
+    existing_question: QuestionModel = await get_question_by_id(question_id)
+    print(existing_question, question_id)
```

Comment:
Suggestion to change the log statement to a more descriptive one along the lines of "Retrieved <existing_question> from database" so it is easier for debugging purposes in the future.